tests/models/gemma/test_modeling_flax_gemma.py

```diff
@@ -143,6 +143,9 @@ def check_use_cache_forward_with_attn_mask(self, model_class_name, config, input
         max_decoder_length = 20
         model = model_class_name(config)
 
+        # make full attn mask since below we are preparing position ids assuming it's all ones
+        attention_mask = jnp.ones_like(attention_mask)
```

Comment:
the other way around: we should update the creation of `position_ids` (below) to match the mask

The same comment applies to other FLAX test changes